src/models/multiBarChart.js

```diff
@@ -179,11 +179,6 @@ nv.models.multiBarChart = function() {
             if (showLegend) {
                 legend.width(availableWidth - controlWidth());
 
-                if (multibar.barColor())
-                    data.forEach(function(series,i) {
-                        series.color = d3.rgb('#ccc').darker(i * 1.5).toString();
-                    });
```

Comment:
At least I would make that the default... one could also override the legend by doing chart.legend.color() I think